radio/src/translations/cn.h.txt

```diff
@@ -63,7 +63,7 @@
 #define TR_TRNCHN                      "CH1CH2CH3CH4"
 
 #define LEN_AUX_SERIAL_MODES           "\010"
-#define TR_AUX_SERIAL_MODES            "调试\0   " "回传镜像" "回传输入" "SBUS教练" "LUA脚本\0""CLI\0     ""GPS\0     ""Debug\0   "
+#define TR_AUX_SERIAL_MODES            "调试\0   " "回传镜像" "回传输入" "SBUS教练" "LUA脚本\0""CLI\0     ""GPS\0     ""Debug\0   ""SpaceMouse"
```

Comment:
8 chars instead of 10 ???

Reply:
Hmm... LEN_AUX_SERIAL_MODES is 10? What did I muss

Reply:
Isn't that octal?